src/client/clientmap.h

```diff
@@ -153,8 +180,9 @@ class ClientMap : public Map, public scene::ISceneNode
 	f32 m_camera_fov = M_PI;
 	v3s16 m_camera_offset;
 
-	std::map<v3s16, MapBlock*> m_drawlist;
+	std::map<v3s16, MapBlock*, MapBlockComparer> m_drawlist;
 	std::map<v3s16, MapBlock*> m_drawlist_shadow;
+	bool m_needsUpdateDrawList;
```

Comment:
this should be named `m_needs_update_drawlist`

Reply:
Renamed